common/utils.go

```diff
@@ -41,7 +42,10 @@ func GetInternalTags() map[string]string {
 		internalTags["application_idx"] = fmt.Sprint(app.Idx)
 		internalTags["application_name"] = app.Name
 	} else {
-		logger.Printf("[ERROR] %v", err)
+		Logger.Printf("[ERROR] %v", err)
 	}
 	return internalTags
 }
+
+var Logger = log.New(os.Stdout, "[WAVEFRONT] ", 0)
```

Comment:
Could we use logrus instead for logging?

Reply:
the nozzle api needs a *log.Logger